encrypted-media-respec.html

```diff
@@ -2547,7 +2557,12 @@
 
         <dt>attribute EventHandler onencrypted</dt>
         <dd>
-          <p>Event handler for the <a def-id="encrypted"></a> event MUST be supported by all HTMLMediaElements as both a content attribute and an IDL attribute.</p>
```

Comment:
It's not implied, I think that https://html.spec.whatwg.org/#event-handlers-on-elements,-document-objects,-and-window-objects is the important bit, normatively. I guess your job might actually be easier if you go back to `GlobalEventHandlers` because then you can just copy that wording.